x-pack/plugins/enterprise_search/server/lib/enterprise_search_config.test.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mockReadFileSync } from '../__mocks__/fs.mock';
+
+import http from 'http';
+import https from 'https';
+
+import { EnterpriseSearchConfig, config } from './enterprise_search_config';
+
+// -----------------------------------------------------------------------------
```

Comment:
Stylistic nit - this is not really something we use in our codebase and I'd prefer not to introduce them as a one-off. Can we remove these from the newly added code?